src/HPWH.hh

```diff
@@ -25,8 +27,57 @@ class RegularGridInterpolator;
  * compiled library.  */
 
 #include "HPWHversion.hh"
+#include "courier/helpers.h"
 
-class HPWH
+class Logger : public Courier::Courier
```

Comment:
I think we should do something very similar to what we are doing in Btwxt for the Default Courier. The idea is that it should be very simple, and it really should be replaced by the caller (e.g., CSE), and the messages should encourage that.

Reply:
This still needs to be addressed.